src/test/java/org/openstreetmap/atlas/checks/validation/tag/InvalidAccessTagCheckTestRule.java

```diff
@@ -0,0 +1,372 @@
+package org.openstreetmap.atlas.checks.validation.tag;
+
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.utilities.testing.CoreTestRule;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Area;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Edge;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Line;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Loc;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Node;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Relation;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Relation.Member;
+
+/**
+ * Tests for {@link InvalidAccessTagCheck}
+ *
+ * @author bbreithaupt
+ */
+
+public class InvalidAccessTagCheckTestRule extends CoreTestRule
+{
+    private static final String TEST_1 = "0.0,0.0";
+    private static final String TEST_2 = "0.1,0.1";
+    private static final String TEST_3 = "0.0,0.2";
+    private static final String TEST_4 = "0.1,0.3";
+    private static final String TEST_5 = "0.0,0.4";
+    private static final String TEST_6 = "-0.1,0.5";
+    private static final String TEST_7 = "0.0,0.6";
+    private static final String TEST_8 = "0.0,0.7";
+    private static final String TEST_9 = "0.1,0.8";
+    private static final String TEST_10 = "0.0,0.9";
+    private static final String TEST_11 = "0.2,-0.1";
+    private static final String TEST_12 = "-0.2,-0.1";
+    private static final String TEST_13 = "-0.2,1.0";
+    private static final String TEST_14 = "0.2,1.0";
```

Comment:
Although it is OK to use these values, I'd suggest using locations that are closer to real life lat/long values.